common/notification/index.ts

```diff
@@ -600,22 +600,6 @@ export async function _actionTakenAt<ID extends ActionID>(
             }
         }
 
-        // --------------- Send out Notifications to send directly --------------------------------------
```

Comment:
Why did you remove this whole block? 🤔  Let's bring it back and fix the parts that use the `verification` field.